integration_tests/sharing_interactions.spec.ts

```diff
@@ -157,4 +157,73 @@ describe('Users sharing data', () => {
     expect(received.notifications[0].relatedObject?.itemPaths[0].bucketKey).not.to.be.null;
     expect(received.notifications[0].relatedObject?.keys[0]).not.to.be.null;
   }).timeout(TestsDefaultTimeout);
+
+  it('users can receive sharing notifications subscription events', async () => {
+    const { user: user1 } = await authenticateAnonymousUser();
+    const { user: user2 } = await authenticateAnonymousUser();
+    const user1Pk = Buffer.from(user1.identity.public.pubKey).toString('hex');
+    const user2Pk = Buffer.from(user2.identity.public.pubKey).toString('hex');
+
+    const txtContent = 'Some manual text should be in the file';
+
+    const storage1 = new UserStorage(user1, TestStorageConfig);
+    const uploadResponse = await storage1.addItems({
+      bucket: 'personal',
+      files: [
+        {
+          path: 'top.txt',
+          data: txtContent,
+          mimeType: 'plain/text',
+        },
+      ],
+    });
+
+    await new Promise<AddItemsEventData>((resolve) => {
+      uploadResponse.once('done', resolve);
+    });
+
+    await storage1.initMailbox();
+
+    const storage2 = new UserStorage(user2, TestStorageConfig);
+    await storage2.initMailbox();
+
+    const ee = await storage2.notificationSubscribe();
+
+    const eventData = new Promise<NotificationSubscribeEventData>((resolve) => {
+      ee.once('data', (d:NotificationSubscribeEventData) => resolve(d));
+    });
+
+    // share with new user
+    const shareResult = await storage1.shareViaPublicKey({
+      publicKeys: [{
+        id: 'new-space-user@fleek.co',
+        pk: user2Pk,
+      }],
+      paths: [{
+        bucket: 'personal',
+        path: '/top.txt',
+      }],
+    });
+
+    const data = await eventData;
+    expect(shareResult.publicKeys).not.to.be.empty;
+    expect(shareResult.publicKeys[0].type).to.equal(ShareKeyType.Existing);
+    expect(shareResult.publicKeys[0].pk).not.to.be.empty;
+
+    const received = await storage2.getNotifications();
```

Comment:
It doesn't like we need this `getNotifications()` call here, since the data is already gotten from the event listener, so we should get rid of it.

Reply:
Fixed this and the linter warnings in [9b4840b](https://github.com/FleekHQ/space-sdk/pull/49/commits/9b4840b9bed05f25bced0497a459034dd61a123a).